### PR TITLE
 Upgrade storybook

### LINKED ISSUE #504: Upgrade storybook
Storybook is released in 6.0 version, and life will be a lot easier if we upgrade now.

### PR DESCRIPTION
**Pull request by wenche**
resolve #504

### CODE REVIEW

**Comment by vnys** [comment]
> There is a warning in console due to [hoisted CSF annotations ](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#hoisted-csf-annotations).

Fixed now

File: apps/storybook-react/stories/Accordion/Accordion.docs.mdx [hunk 1]
```diff
@@ -39,8 +41,7 @@ const {
 
 ```jsx
 import { Accordion } from '@equinor/eds-core-react'
-
-<Accordion>
+;<Accordion>
```
Comment (vnys) [inline apps/storybook-react/stories/Accordion/Accordion.docs.mdx:44]:
```suggestion
<Accordion>
```

**Review by vnys — APPROVED** [review]
Just a semi colon that had snuck in somehow, apart from that LGTM 👍

File: apps/storybook-react/stories/Accordion/Accordion.docs.mdx (hunk 1 above)
Comment (wenche) [reply apps/storybook-react/stories/Accordion/Accordion.docs.mdx:44]: It's simply not possible for me to save without I guess prettier adding that semicolon :/

**Pull request merged**